.vitepress/config.ts

```diff
@@ -35,26 +35,26 @@ const versionLinks = ((): DefaultTheme.NavItemWithLink[] => {
     case 'local':
       return [
         {
-          text: 'Vite 4 Docs (release)',
+          text: 'Vite 4 ドキュメンテーション (リリース)',
```

Comment:
サイト内を確認したところ、「ドキュメンテーション」という表現は使われていなかったので、「ドキュメント」に変更お願いします:pray:
（ついでにカッコを全角に）
```suggestion
          text: 'Vite 4 ドキュメント（リリース）',
```